### ISSUE TITLE
Replace broken link with regular link

### ISSUE THREAD

**Issue by johnshaughnessy** [mozilla/hubs#3698]
In https://github.com/mozilla/hubs/issues/1747 we decided to change the image that appears when we fail to find an image, video, or 3D model to associate with a link. 

What if instead of showing a "broken" media link, we fall back to a generic "link" image. People can still interact with the content (pin it, open it in another tab, etc). It is still "broken" if we fail media resolution, and people can click "refresh" to force the backend to retry, but if there are cases that will _always_ fail, maybe it's better to just default to show a (non-broken) "link to external resource".

**Comment by misslivirose** [comment]
Are you thinking that the visual would just be a chance in visuals to the image we're already displaying, or an update to functionality when we fail to resolve a media element? We're also going to be looking at https://github.com/mozilla/Spoke/issues/1049 this month which will also improve how users can make a link appear in their space. 

I can understand both sides - on the one hand, if I add a link and it doesn't resolve when I expect it to, it might be an error I want to try and resolve myself (e.g. a typo) and in that case, I'd assume it was fine if the image didn't indicate otherwise.

**Comment by johnshaughnessy** [comment]
Yes - I am only suggesting a change to the visual asset that appears in cases that we do not resolve an image/video/model for the provided link. 

Participants have little recourse if their links fail to resolve as expected. It seems better (to me) to have something that does not look broken (especially if the link is embedded in a scene). The fallback object is a functioning link, so I think the visual should indicate what it is and does rather than highlighting that something (may have) gone wrong. 

This might require us to increase the discoverability of the intended (resolve-to-media) behavior is so that new users are encouraged to try clicking "refresh" at least once if a model/image/video does not appear as expected when they paste a link.

I am not very concerned with typos specifically because I view this similarly to having links with typos in markdown. (e.g. if instead of [hubs.mozilla.com](hubs.mozilla.com) I type [hups.mozilla.com](hups.mozilla.com), the ladder still looks like a link despite being incorrect. It's up to me to verify that it points to what I want.)

The link enhancement suggestions (https://github.com/mozilla/Spoke/issues/1049 ) look good to me. If the image tied to an "enhanced link" (as described in the linked issue) failed to load, then the "broken link" image seems appropriate to me again.
